### PR TITLE
doc/saul: add cross link between SAUL and SAUL registry

### PR DESCRIPTION
**Pull request by mali**
fix issue #6631 
tested by generating pdf doc from latex files.

### CODE REVIEW

File: sys/include/saul_reg.h [hunk 1]
```diff
@@ -11,11 +11,13 @@
  * @ingroup     sys
  * @brief       Global sensor/actuator registry for SAUL devices
  *
+ * See also \ref drivers_saul
  * @{
  *
  * @file
  * @brief       SAUL registry interface definition
  *
+ *
```
Comment (gebart) [inline sys/include/saul_reg.h]: You can remove the extra newline

File: sys/include/saul_reg.h [hunk 2]
```diff
@@ -11,11 +11,13 @@
  * @ingroup     sys
  * @brief       Global sensor/actuator registry for SAUL devices
  *
+ * See also \ref drivers_saul
```
Comment (gebart) [inline sys/include/saul_reg.h]: You should be able to use @ instead of \ like for the other doxygen commands, to match the coding style of RIOT.

Comment (gebart) [inline sys/include/saul_reg.h]: There is also a command `@see` which adds a "see also" paragraph in the docs.

**Comment by mali** [comment]
@gebart : done
yes, I've checked html doc, it's ok

**Pull request merged**